ui/src/containers/AlertProvider.js

```
@@ -0,0 +1,32 @@
+import React, { createContext, useContext } from 'react';
+import { useQuery } from 'react-query';
+import { getAlerts } from '../services/alertmanager/api';
+import { filterAlerts } from '../services/alertUtils';
+
+const AlertContext = createContext(null);
+
+export function useAlerts(filters: FilterLabels) {
+  const query = useContext(AlertContext);
+  if (query.status === 'success') {
```

I think a null check on query is missing here and should throw an exception to indicate that this hook can only be used within an `AlertProvider`